lib/bitmap.h

```diff
@@ -104,6 +104,11 @@ M0_INTERNAL size_t m0_bitmap_ffz(const struct m0_bitmap *map);
  */
 M0_INTERNAL void m0_bitmap_set(struct m0_bitmap *map, size_t idx, bool val);
 
+/**
+   Reset a bitmap.
+ */
```

Comment:
It's better to have,
```
/**
 *  Reset a bitmap.
 */
```

Reply:
I just didn't want to mix different styles in the same file.